resources/zh/cemu.po

```diff
@@ -222,20 +222,20 @@ msgstr "Cemu的CPU使用率 %"
 
 #: GeneralSettings2.cpp:221
 msgid "CPU per core usage"
-msgstr "CPU每核心使用率"
+msgstr "CPU各核心使用率"
 
 #: GeneralSettings2.cpp:222
 #, c-format
 msgid "Total cpu usage in % for each core"
-msgstr "每个核心的总CPU使用率 %f"
+msgstr "各CPU核心分别的使用率 %f"
```

Comment:
'%f' was definitely put to silence Poedit errors about that string. That's because the English .pot has an error which tells Poedit to process the '% f' as a C formatting specifier.

Reply:
Two options: '各CPU核心分别的使用率' or '各CPU核心以%计的分别的使用率'. 
I don't know about format, I'm sorry.

Reply:
Current is fine, no need to change that.

Reply:
OK, thanks.